src/compiler/watch.ts

```diff
@@ -220,7 +233,17 @@ namespace ts {
         }
 
         function emitFilesAndReportErrorUsingBuilder(builderProgram: BuilderProgram) {
-            emitFilesAndReportErrors(builderProgram, reportDiagnostic, writeFileName);
+            const compilerOptions = builderProgram.getCompilerOptions();
+            const reportSummary = (errorCount: number) => {
+                if (errorCount === 1) {
+                    onWatchStatusChange(createCompilerDiagnostic(Diagnostics.Found_1_error, errorCount), sys.newLine, compilerOptions);

```

Comment:
The new line character here used is incorrect, You dont want to use newline from ```sys``` but it should depend on compiler options and host.getNewLine()